packages/components/src/components/ScalarSearchResults/ScalarSearchResultItem.vue

```diff
@@ -23,21 +23,23 @@ const attrs = computed(() => {
       v-bind="attrs.rest"
       :class="
         cx(
-          'group flex cursor-pointer gap-3 rounded px-3 py-1.5 no-underline hover:bg-b-2',
+          'group flex cursor-pointer gap-2.5 rounded px-3 py-1.5 no-underline hover:bg-b-2',
           {
             'bg-b-2': active,
           },
           attrs.className,
         )
       ">
       <!-- Icon -->
-      <div class="text-c-3 group-hover:text-c-1">
+      <div
```

Comment:
This is just to line up the icon with the first line?

Reply:
@cameronrohani

Reply:
feel free to do it differently but needs to be aligned with first line of text no matter the font size